src/Pim/Bundle/VersioningBundle/Resources/config/guessers.yml

```diff
@@ -4,7 +4,6 @@ parameters:
     pim_versioning.update_guesser.chained.class:           Pim\Bundle\VersioningBundle\UpdateGuesser\ChainedUpdateGuesser
     pim_versioning.update_guesser.contains_products.class: Pim\Bundle\VersioningBundle\UpdateGuesser\ContainsProductsUpdateGuesser
     pim_versioning.update_guesser.variant_group.class:     Pim\Bundle\VersioningBundle\UpdateGuesser\VariantGroupUpdateGuesser
-    pim_versioning.update_guesser.product_value.class:     Pim\Bundle\VersioningBundle\UpdateGuesser\ProductValueUpdateGuesser
```

Comment:
From 1.7 changelog:
`- Remove useless service and parameter: `pim_enrich_image` and `pim_enrich.form.type.image.class`
`
So I would say: note it all!